test/testutil/constants.go

```diff
@@ -1,7 +1,14 @@
 package testutil
 
+import crdanno "github.com/operator-framework/rukpak/cmd/crdvalidator/annotation"
+
 const (
 	DefaultCrdName = "samplecrd"
 	DefaultCrName  = "samplecr"
 	DefaultGroup   = "e2e.io"
 )
+
+var (
+	CrdValidationEnabled  = map[string]string{crdanno.ValidationKey: crdanno.Enabled}
+	CrdValidationDisabled = map[string]string{crdanno.ValidationKey: crdanno.Disabled}
```

Comment:
Do we use either of these variables anywhere? I wasn't able to find their usage throughout this diff.

Reply:
We do not, these are left over from previous iterations. Will remove.